platform/api.search/src/org/netbeans/modules/search/matcher/BufferedCharSequence.java

```diff
@@ -170,12 +167,9 @@ private BufferedCharSequence(BufferedCharSequence bufferedCharSequence) {
     @Override
     @SuppressWarnings("FinalizeDeclaration")
     protected void finalize() throws Throwable {
-        try {
```

Comment:
One of the features of [frgaal](http://frgaal.org) is that it can report _"future deprecations"_ like [here](https://github.com/apache/netbeans/actions/runs/4657374401/jobs/8241911775?pr=4795#step:6:8204) - the compiler contains list of all deprecated methods up to JDK 20 and can emit warnings for each of them. When the module is compiled with `-Werror` one has to fix such _"future warnings"_ or `@SuppressWarnings` them.